test/contracts/replayprotection.test.ts

```diff
@@ -0,0 +1,636 @@
+import "mocha";
```

Comment:
Name of the file should be replayProtection.test.ts

Reply:
it doesnt let me fix file name properly on this mac mmm we will need t osirt it somehow lol